packages/angular-material/src/other/table.renderer.ts

```diff
@@ -48,12 +48,33 @@ import {
 @Component({
   selector: 'TableRenderer',
   template: `
+
```

Comment:
Please remove this superfluous empty line

Reply:
Done